src/website/lib/redux/rootReducer.ts

```diff
@@ -17,6 +18,7 @@ export function rootReducer() {
     batteries: new BatteriesReducer().reducer,
     windSensors: new WindSensorsReducer().reducer,
     genericSensors: new GenericSensorsReducer().reducer,
+    timestamp: new TimestampReducer().reducer,
```

Comment:
Can we change this to `timestamp` -> `dataFilter`? Just so the name is more descriptive  

I want to think of what we're storing here as a general filter (which includes timestamp, but can include other inputs in the future as well)

Reply:
done